### PR TITLE
Update libsecret to 0.21.3

### PR DESCRIPTION
**Pull request by ZVNexus**

### CODE REVIEW

File: libsecret/libsecret.json [hunk 1]
```diff
@@ -2,6 +2,7 @@
   "name": "libsecret",
   "buildsystem": "meson",
   "config-opts": [
+    "-Dcrypto=gnutls",
```
Comment (bbhtt) [inline libsecret/libsecret.json:5]: This needs gnutls >= 3.8.2, the runtime ships 3.8.0 it silently falls back to libgcrypt.

Comment (bbhtt) [reply libsecret/libsecret.json:5]: https://gitlab.com/freedesktop-sdk/freedesktop-sdk/-/merge_requests/17826

Comment (A6GibKm) [reply libsecret/libsecret.json:5]: Please propose this change in a separate MR.

Comment (ZVNexus) [reply libsecret/libsecret.json]: > Please propose this change in a separate PR.

Done

**Pull request merged**